scripts/gen-auth-api-spec.ts

```diff
@@ -130,6 +128,70 @@ async function toOpenapi3(discovery: any): Promise<any> {
   return openapi3;
 }
 
+const pathParamsForFlatPathParam: Map<string, string> = new Map([
```

Comment:
I *think* the type can be inferred here (if it can, remove the explicit typing), otherwise no worries

go/tsstyle#type-inference